src/test/java/org/mitre/synthea/export/ExporterTest.java

```diff
@@ -52,11 +52,9 @@ public void setup() throws Exception {
     Provider.loadProviders(location, ProviderTest.providerRandom);
     record = patient.record;
     // Ensure Person's Payer is not null.
-    Payer.loadNoInsurance();
-    for (int i = 0; i < age; i++) {
-      long yearTime = time - years(i);
-      patient.coverage.setPayerAtTime(yearTime, Payer.noInsurance);
-    }
+    PayerManager.loadPayers(new Location("Massachusetts", null));
```

Comment:
Ideally, you do not want to specify a location in the unit tests like "Massachusetts" because it will break synthea-international support. Instead, you want to do something like

```
String testStateDefault = Config.get("test_state.default", "Massachusetts");
PayerManager.loadPayers(new Location(testStateDefault, null));
```